controllers/kubeletconfig_controller.go

```diff
@@ -231,10 +209,86 @@ func (r *KubeletConfigReconciler) syncConfigMap(ctx context.Context, mcoKc *mcov
 	return rendered, nil
 }
 
+func (r *KubeletConfigReconciler) makeKCHandlerForPlatform(ctx context.Context, instance *nropv1.NUMAResourcesOperator, kcKey client.ObjectKey) (*kubeletConfigHandler, error) {
```

Comment:
not completely happy about this looks, but I prefer to merge this version which looks reasonnably good, then (add tests and then) improve later.